cmake/modules/FindRapidJSON.cmake

```diff
@@ -8,6 +8,49 @@
 # RapidJSON_FOUND - system has RapidJSON parser
 # RapidJSON_INCLUDE_DIRS - the RapidJSON parser include directory
 #
+if(ENABLE_INTERNAL_RapidJSON)
+  include(ExternalProject)
+  file(STRINGS ${CMAKE_SOURCE_DIR}/tools/depends/target/rapidjson/Makefile VER)
+  string(REGEX MATCH "VERSION=[^ ]*" RJSON_VER "${VER}")
+  list(GET RJSON_VER 0 RJSON_VER)
+  string(SUBSTRING "${RJSON_VER}" 8 -1 RJSON_VER)
```

Comment:
please use 
>file(STRINGS ${CMAKE_SOURCE_DIR}/tools/depends/target/rapidjson/Makefile VER REGEX "^[ ]*VERSION[ ]*=.+$")
string(REGEX REPLACE "^[ ]*VERSION[ ]*=[ ]*" "" RJSON_VER "${VER}")

It is much more tolerant to whitespace and version formating changes. And less code ;)

Reply:
I literally copied it from FindCrossGUID.cmake, I'll change it and squash commits, no problem with that.

Maybe do the same for other cmakes in a separate PR ? :)

Reply:
@wsnipex Your REGEX needed a small tweak to make it work, but it's pushed already :) Thanks for the comments and feedback.

Reply:
if you mean the '*', then it is there in my comment, but stupid markdown ate it :)